marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/scenarios.modeling.tsx

```diff
@@ -30,6 +30,87 @@ describe('Entity Modeling', () => {
       cy.resetTestUser();
   });
 
+  it('can add new properties to existing Entity, and revert the entity', () => {
```

Comment:
Could you add this test in front of one of the current tests? For e2e testing, I think it's better to have longer tests that simulate real user interactions.

Reply:
Updated tests.